frameworks/helloworld/tests/test_region_awareness.py

```diff
@@ -14,18 +14,38 @@
 POD_NAMES = ['hello-0', 'world-0', 'world-1']
 
 
-@pytest.mark.dcos_min_version('1.11')
-@pytest.mark.region_awareness
-@sdk_utils.dcos_ee_only
-def test_nodes_deploy_to_local_region_by_default():
+@pytest.fixture
+def local_service():
     sdk_install.install(
```

Comment:
We do need the `try - finally` block here so that uninstall is run even if the install fails.